packages/ui/components/core/Button.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import { Icon } from '@iconify/react'
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+
+import React from 'react'
+
+import { Button, Center } from '@mantine/core'
+
+export default {
+	title: 'App/Layout/eButton',
+	component: Button,
+	decorators: [
+		(Story) => (
+			<Center style={{ width: '100vw' }}>
+				<Story />
+			</Center>
+		),
+	],
+	parameters: {
+		design: {
+			type: 'figma',
+			url: 'https://www.figma.com/file/gl8ppgnhpSq1Dr7Daohk55/Design-System-(2023)?node-id=196%3A5045&t=0SZ0JVMYe5r7bNkb-4',
+		},
+	},
+} as ComponentMeta<typeof Button>
+
+// const Large: ComponentStory<typeof Button> = (args) => (
+// 	<Button leftIcon={<Icon icon={args.icon} />} {...args} />
+// )
+// const Small: ComponentStory<typeof Button> = (args) => <Button {...args} />
+
+const ButtonVariant: ComponentStory<typeof Button> = (args) => <Button {...args} />
+
+export const SmallPrimary = ButtonVariant.bind({})
+export const SmallSecondary = ButtonVariant.bind({})
+export const SmallAccent = ButtonVariant.bind({})
+export const LargePrimary = ButtonVariant.bind({})
+export const LargeSecondary = ButtonVariant.bind({})
+export const LargeAccent = ButtonVariant.bind({})
+
+SmallPrimary.args = {
+	variant: 'sm-primary',
+	children: 'Download the app',
+}
+SmallSecondary.args = {
+	variant: 'sm-secondary',
+	children: 'InReach.org',
+}
+SmallAccent.args = {
+	variant: 'sm-accent',
+	children: 'Safety exit',
+}
+LargePrimary.args = {
+	variant: 'lg-primary',
+	children: 'More sorting options',
+	// icon: 'lucide:sliders',
+	leftIcon: <Icon icon='lucide:sliders' />,
+}
+LargeSecondary.args = {
+	variant: 'lg-secondary',
+	children: 'Save',
+	// icon: 'fe:heart-o',
+	leftIcon: <Icon icon='fe:heart-o' />,
+}
+LargeAccent.args = {
+	variant: 'lg-accent',
+	children: 'View map',
+	// icon: 'fe:map',
+	leftIcon: <Icon icon='fe:map' />,
```

Comment:
Just for future reference - since it's all based off the same component, you can just define a single base story and run the variants off of that.
I move the `leftIcon` prop down to the args because TypeScript wasn't happy with the `icon` prop.